app/code/Magento/Pwa/Test/Mftf/Section/VeniaCartPageSection.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<sections xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/SectionObject.xsd">
+    <section name="VeniaCartPageSection">
+        <element name="cartHeader" type="text" selector="//h1[contains(text(), 'Cart')]"/>
+        <element name="emptyCartMessage" type="text" selector="//h3[contains(text(), 'There are no items in your cart.')]"/>
+        <element name="signIn" type="button" selector="[class*='button-content-']"/>
+        <element name="checkout" type="button" selector="//button[contains(@class, 'button-root_highPriority-')]/span[contains(text(), 'Proceed to Checkout')]"/>
+    </section>
+
+    <section name="CartPageShippingSection">
+        <element name="shippingAccordion" type="button" selector="//button[contains(@class, 'section-title_container-')]/div[contains(text(), 'Estimate your Shipping')]"/>
+    </section>
+
+    <section name="CartPageCouponCodeSection">
+        <element name="couponCodeAccordion" type="button" selector="//button[contains(@class, 'section-title_container-')]/div[contains(text(), 'Enter Coupon Code')]"/>
+    </section>
+
+    <section name="CartPageGiftCardSection">
+        <element name="giftCardAccordion" type="button" selector="//button[contains(@class, 'section-title_container-')]/div[contains(text(), 'Apply Gift Card')]"/>
+    </section>
+
+    <section name="CartPageGiftOptionsSection">
+        <element name="giftOptionsAccordion" type="button" selector="//button[contains(@class, 'section-title_container-')]/div[contains(text(), 'See Gift Options')]"/>
+    </section>
+
+    <section name="CartPageProductDetailsSection">
+        <element name="cartProductName" type="text" selector="//span[contains(text(), '{{var_product_name}}')]" parameterized="true"/>
+    </section>
+
+    <!--<section name="CartPageQtySection">-->
+
+    <!--</section>-->
+
+    <section name="CartPageKebabSection">
+        <element name="kebabMenu" type="button" selector="//li[./div/span[contains(text(), '{{product_name}}')]]//button[contains(@class, 'kebab-kebab-')]" parameterized="true"/>
+        <element name="editItem" type="button" selector="(//*[contains(@class, 'kebab-dropdown_active-')]/*[contains(@class, 'section-menuItem-')])[2]"/>
+        <element name="removeItem" type="button" selector="(//*[contains(@class, 'kebab-dropdown_active-')]/*[contains(@class, 'section-menuItem-')])[3]"/>
```

Comment:
The `[2]` / `[3]` seems very fragile. Is there any other way to target these?

Reply:
I had tried other options like contains(text(), 'Edit item')' but they complain about wrong element receiving click events. Its already running in `Section/VeniaShoppingCartSection` >  Section `VeniaShoppingCartKebabSection`.  For Cart page, I just copied them to have separated in case CSS changes in future.